packages/investor/src/pages/programs/program-details/components/notify-button.tsx

```diff
@@ -36,14 +36,14 @@ class _NotifyButton extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const { t } = this.props;
+    const { t, disabled } = this.props;
     return (
       <div className="notify-button">
         <GVButton
           className="program-details-description__invest-btn"
           onClick={this.handleClick}
           disabled={Boolean(
-            this.state.notificationId || this.state.subscription
+            this.state.notificationId || this.state.subscription || disabled
```

Comment:
Move disabled condition outside the component

Reply:
Moved the disabled condition inside the component NotifyButton